utils/generateErrorfile.py

```diff
@@ -64,15 +64,15 @@ def genCFile(errors, filename):
 
   headerfile = os.path.splitext(filename)[0] + ".h"
 
-  impl = prologue\
-         + "#include \"Basics/Common.h\"\n"\
-         + "#include \"Basics/error.h\"\n"\
-         + "#include \"Basics/voc-errors.h\"\n"\
-         + "\n"\
-         + "/// helper macro to define an error string\n"\
-         + "#define REG_ERROR(id, label) TRI_set_errno_string(TRI_ ## id, label);\n"\
-         + "\n"\
-         + "void TRI_InitializeErrorMessages() {\n"
+  impl = prologue + """
+#include "Basics/error.h"
+#include "Basics/voc-errors.h"
+
+/// helper macro to define an error string
+#define REG_ERROR(id, label) TRI_set_errno_string(TRI_ ## id, label);
+
+void TRI_InitializeErrorMessages() {
```

Comment:
Suggestion: instead of having a separate initialization function and registering each error message at program start, can we switch to a compile-time map of error codes to error messages?
`frozen::map` should be a good fit, I hope.

Reply:
Love that idea, will try it out.

Reply:
https://github.com/arangodb/arangodb/pull/13496